libs/langchain/Makefile

```diff
@@ -76,6 +76,7 @@ lint format: PYTHON_FILES=.
 lint_diff format_diff: PYTHON_FILES=$(shell git diff --relative=libs/langchain --name-only --diff-filter=d master | grep -E '\.py$$|\.ipynb$$')
 
 lint lint_diff:
+	./scripts/check_pydantic.sh
```

Comment:
Consider making this take a path to scan? Otherwise a violation in `experimental` might block `langchain` or vice versa, which doesn't seem great.

Reply:
oh and also should we add this to libs/experimental? and move script to top level

Reply:
@obi1kenobi langchain and experimental have each their own Makefile right now. This PR only adds the linter to langchain, but not to experimental. 

I'd rather keep the scripts independent for langchain and experimental for now -- not worth generalizing for a single script given that it breaks pattern of linting. I can follow up with another PR to add the same code to experimental.

Reply:
if you run the script in root of repo it'd catch things in experimental, right? but i guess if you're only accessing via `make lint` then you'll be in `libs/langchain` so should be fine?

Reply:
i misunderstood the ask ^ added a path argument